wafer/pages/models.py

```diff
@@ -22,7 +22,11 @@ class File(models.Model):
     item = models.FileField(upload_to='pages_files')
 
     def __str__(self):
-        return u'%s' % (self.name,)
+        if self.pages.exists():
```

Comment:
I'm wondering how sane it is to have `__str__` make database calls and whether this should rather be a separate method that is then explicitly specified in the admin interface?